### DISCUSSION TITLE
Glowroot vs OpenTelemetry

### DISCUSSION THREAD

**Discussion by daveram** [glowroot/glowroot#929] [General]
We've been happily using Glowroot for a couple years now, shout out to a great project that works!

That said, we've recently been looking at other APM tools as we're starting to use python apps in addition to our java-based apps.  Noticed that @trask you've been working on OpenTelemetry which supports many languages, are you aware of a breakdown on the differences between Glowroot vs OpenTelemetry?  I'm assuming there's a lot of feature parity here and that's why the focus is now more on OpenTelemetry

For example, I can't find any information on how slow traces are captured in OpenTelemetry like they are in Glowroot

**Comment by garethedwards-tass** [comment]
No question, out of the box, Glowroot provides greater simplicity and features that are hard / near impossible to find in Opentelemtry and most OSS APM front ends, I've tried them all... currently having a look at HyperDX which was bought by click house.

What I'd like to know is.... why ? and what others experiences are

Glowroot seems to provide the following out of the box that seems to be a stretch for most other APM's, am I missing something, or is this just how it is ?

Simple drill downs into data
Grouping / consolidation of information
Database selects, grouped by statements, records returned ??
